src/shortcuts.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+'use strict';
+
+import {
+  IShortcutItem
+} from 'phosphide';
+
+export
+const SHORTCUTS: IShortcutItem[] = [
+  {
+    command: 'command-palette:activate',
+    args: void 0,
+    selector: '*',
+    sequence: ['Alt Space']
+  },
+  {
+    command: 'command-palette:deactivate',
+    args: void 0,
+    selector: '*',
+    sequence: ['Escape']
+  },
+  {
+    command: 'file-operations:new-text-file',
+    args: void 0,
+    selector: '*',
+    sequence: ['Ctrl O']
+  },
+  {
+    command: 'file-operations:new-notebook',
+    args: void 0,
+    selector: '*',
+    sequence: ['Ctrl Shift N']
+  },
+  {
+    command: 'file-operations:save',
+    args: void 0,
+    selector: '.jp-Document.jp-mod-focus',
```

Comment:
The focus selector is redundant (it will have focus in order to get a keyboard event), and will be going way.